src/dom/README.md

```diff
@@ -264,3 +264,140 @@ assert.isTrue(string === "" + ionString);
 assert.isTrue(string === ionString.stringValue());
 assert.isFalse(ionString === ionString2);
 ```
+
+### `dom.Value` equivalence
+
+dom.Value object equivalence is implemented by two methods ionEquals(compares two dom.Value objects based on strict or non-strict comparison type) and equals(compares a dom.Value object with a corresponding JS object).
+
+#### `ionEquals`:
+
+This method compares a dom.Value object with another dom.Value object with either strict or non-strict comparison type. 
+
+*Strict equivalence* refers to Ion data model equivalence as defined in Ion Equivalence (https://www.javadoc.io/doc/com.amazon.ion/ion-java/latest/com/amazon/ion/util/Equivalence.html) and by Ion Specification (http://amzn.github.io/ion-docs/docs/spec.html)
+*Structural or non-strict equivalence* follows the same rules as strict equivalence,except that
+
+* 1. Annotations are not considered, and
+* 2. Timestamps that represent the same instant in time are always considered equivalent.
+
+#### `equals`:
+
+This method compares a dom.Value object with a corresponding JS object. 
```

Comment:
I think you should go into a bit more detail here.
* How are annotations handled? (I think they're ignored, right?) 
* For Ion types that don't have an obvious mapping to JS, what types can be considered equal? 
   * Is an `ion.dom.Blob` equal to an `Array`? A `Utf8Array`? A `Buffer`?)
   * Can I test an `ion.dom.Decimal` for equality with a `Number`?
   * etc